packages/contracts-bedrock/src/deploy-utils.ts

```diff
@@ -445,9 +448,9 @@ export const doPhase = async (opts: {
     console.log(`Please execute phase ${opts.phase}...`)
     console.log(`MSD address: ${opts.SystemDictator.address}`)
     console.log(`JSON:`)
-    console.log(jsonifyTransaction(tx))
+    console.log(printJsonTransaction(tx))
```

Comment:
why two `console.log` calls? the print function returns `void`

Reply:
missed that, will fix.